src/utils/users.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```

Comment:
Can you please delete this file, its here cause of a failed rebase, after that we can merge this